File: web_mes/src/models/Payment/Reservation.js
export class ReservationModel{
    constructor({id, optionID, paymentID, quantity, endTime}){
        this.id = id;
        this.optionID = optionID;
        this.paymentID = paymentID;
        this.quantity = quantity;
        this.endTime = endTime;
    }
}

File: GolangQuest/internal/order/entities/reservation.go
package entities

import (
	"gorm.io/datatypes"
	"gorm.io/gorm"
)

type Reservation struct {
	gorm.Model
	OptionID uint           `gorm:"column:option_id"`
	OrderID  uint           `gorm:"column:order_id"`
	Quantity int            `gorm:"column:quantity"`
	EndTime  datatypes.Time `gorm:"column:end_time"`
}

func (Reservation) TableName() string {
	return "Reservation"
}
